testsuite/MDAnalysisTests/data/coordinates/create_data.py

```diff
@@ -7,17 +7,19 @@ def create_test_trj(uni, fname):
     n_atoms = uni.atoms.n_atoms
     pos = np.arange(3 * n_atoms).reshape(n_atoms, 3)
     uni.trajectory.ts.dt = 1
+    orig_box = np.array([80, 80, 80, 60, 60, 90], dtype=np.float32)
```

Comment:
I would be worth having a slightly more complex box. At least with different values in each dimension, perhaps with non integer values as well.